src/github/repository/index.ts

```diff
@@ -27,8 +27,9 @@ type RETVAL = {
 export const execute = async (path?: string, spinner?: Ora): Promise<RETVAL | undefined> => {
   let retVal: RETVAL | undefined;
 
-  if (process.env.TRAVIS_REPO_SLUG) {
-    const [owner, repo] = process.env.TRAVIS_REPO_SLUG.split('/') as [string, string];
+  if (process.env.TRAVIS_REPO_SLUG || process.env.REPO_SLUG) {
+    const slug = process.env.TRAVIS_REPO_SLUG || process.env.REPO_SLUG;
+    const [owner, repo] = slug!.split('/') as [string, string];
```

Comment:
Let's retain parity with the logic for Travis and CircleCI by tying into GitHub's [default environment variables](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables). I'd suggest something like...

```
} else if (process.env.GITHUB_ACTIONS) {
  const owner = process.env.GITHUB_REPOSITORY_OWNER;
  const repo = process.env.GITHUB_REPOSITORY.split('/')[1];

  retVal = { owner, repo };
}
```

...or similar. This also clearly sets GH actions as it's own situation.